packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -133,6 +133,7 @@ async function exists(filePath) {
 }
 
 async function getMendixVersion() {
+    const mendixMajorVersion = process.argv[process.argv.indexOf("--mx-major-version") + 1];
```

Comment:
Maybe `mx-version` only?

Reply:
I agree. You can even pass a more specific version to filter even more out, like `8.16`. I will change it.